jlm/mlir/frontend/MlirToJlmConverter.cpp

```diff
@@ -634,6 +662,57 @@ MlirToJlmConverter::ConvertFPSize(unsigned int size)
   }
 }
 
+llvm::linkage
+MlirToJlmConverter::ConvertLinkage(std::string stringValue)
+{
+  if (!stringValue.compare("external_linkage"))
```

Comment:
This code might be better placed in `jlm/llvm/ir/linkage.hpp`, as it is the inverse of the existing `ToString`. Could you create a `FromString` there? You can use an implementation based on a `static std::unordered_map<std::string_view, linkage> = {{"blabla", x}, {"blublu", y}}`.

Reply:
Agree